### DISCUSSION TITLE
Outpainting on lama-cleaner

### DISCUSSION THREAD

**Discussion by ShmuelRonen** [Sanster/IOPaint#327] [Ideas]
First of all, thanks for developing the wonderful Lama cleaner product.

Although the main use is in manipulating images within their basic framework, I found a way to create Outpainting in a simple way.

To do this, insert an image with a white background around it. which allows creation of its extension.

Attached is an example of this 'hidden' ability.

Is it possible to add a built-in option to lama cleaner to create a white background around an image and thus enable outpainting?
![1](https://github.com/Sanster/lama-cleaner/assets/80190186/0c525e1c-fbd1-4634-8ab8-256e9a26b0cf)
![2](https://github.com/Sanster/lama-cleaner/assets/80190186/5f9832c7-9983-45c1-b692-6c8948e0ee56)
![3](https://github.com/Sanster/lama-cleaner/assets/80190186/c5953ea0-390c-4223-a03b-6b379dfbcb01)

**Comment by Sanster** [comment]
Thank you for the example, this usage is very interesting. I also saw that other users want the outpainting feature, but currently, I am focusing on developing my [first macOS app](https://github.com/Sanster/lama-cleaner/discussions/314), if I can come up with a method that requires less workload to implement outpainting, I will add it to Lama Cleaner.
